layouts/_default/footer.html

```diff
@@ -0,0 +1,29 @@
+<footer>
+  <div class="subfooter newsletter">
+    <div id="mc_embed_signup" class="subfooter-content">
+      <span>{{ .Params.newsletter }}</span>
+      <form action="{{ .Params.mailchimp }}" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+        <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="" required>
+        <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_c40666a9e35639e313e69839f_795a94f3ff" tabindex="-1" value=""></div>
+        <button class="btn" type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe">{{ .Params.subscribe }}</button>
+      </form>
+    </div>
+  </div>
+  <div class="subfooter social">
+    <div class="subfooter-content">
+      <div class="sitemap">
+        {{ range .Site.Menus.main }}
+          <a href="{{ .URL }}">{{ i18n .Identifier }}</a>
```

Comment:
To be consistent, we will probably need to add smooth scrolling onclick of these links. Check the ```handleMenuItemClick``` function of the menu.js file to see how it's done for the menu. It should be a lot less complicated since you don't have to manage timeouts (wait for the menu to close before scrolling for example). Code should look something like that:
```
this.sitemapItems = $('footer .sitemap a');
this.sitemapItems.click((e) => {
    const sectionId = $(e.currentTarget).attr('href');
    const sectionOffset = getSectionOffset(sectionId);

    $('html, body').stop().animate({
        scrollTop: sectionOffset
    }, GET_STARTED_ANIMATION_DURATION);

    e.preventDefault();
});
```